internal/pkg/errors/strings.go

```diff
@@ -60,13 +60,7 @@ const (
 	KsqlDBNotBackedByKafkaMsg = "The ksqlDB cluster \"%s\" is backed by \"%s\" which is not the current Kafka cluster \"%s\".\nTo switch to the correct cluster, use `confluent kafka cluster use %s`.\n"
 
 	// local commands
-	AvailableServicesMsg       = "Available Services:\n%s\n"
-	UsingConfluentCurrentMsg   = "Using CONFLUENT_CURRENT: %s\n"
-	AvailableConnectPluginsMsg = "Available Connect Plugins:\n%s\n"
-	StartingServiceMsg         = "Starting %s\n"
-	StoppingServiceMsg         = "Stopping %s\n"
-	ServiceStatusMsg           = "%s is [%s]\n"
-	DestroyDeletingMsg         = "Deleting: %s\n"
+	ConfluentLocalThankYouMsg = "Thank you for using Confluent local.\n"
```

Comment:
This seems a bit cheesy, what do you think?

Reply:
idk lol i wanted it to look like some friendly trial?